src/core/include/openvino/runtime/tensor.hpp

```diff
@@ -325,4 +326,18 @@ Tensor read_tensor_data(const std::filesystem::path& file_name,
                         const PartialShape& shape = PartialShape::dynamic(1),
                         std::size_t offset_in_bytes = 0,
                         bool mmap = true);
+
+/// \brief Read a tensor content from an externally provided file handle. Only raw data is loaded.
+/// \param file_handle File handle to read. File handle should be opened by the caller OpenVINO takes ownership of the file
+///                    handle and is responsible for closing it. File should be opened with GENERIC_READ, FILE_SHARE_READ flags on Windows and O_RDONLY on Linux.
+/// \param element_type Element type, when not specified the it is assumed as element::u8.
+/// \param shape Shape for resulting tensor. 
+/// \param offset_in_bytes Read file starting from specified offset.
+/// \param mmap Use mmap that postpones real read from file until data is accessed. Does not support mmap=false mode.
```

Comment:
'Does not support mmap=false mode.' I don't get it .. only default true is allowed?